service/header/local_exchange.go

```diff
@@ -0,0 +1,29 @@
+package header
+
+import "context"
+
+type LocalExchange struct {
```

Comment:
why exported? where will this be used?

Reply:
It can be used by anyone who does not want to deal with networking. Rn useful for testing